seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -2967,7 +2967,7 @@
         "assert($(\"h1\").hasClass(\"pink-text\"), 'message: Your <code>h1</code> element should have the class <code>pink-text</code>.');",
         "assert($(\"h1\").hasClass(\"blue-text\"), 'message: Your <code>h1</code> element should have the class <code>blue-text</code>.');",
         "assert($(\"h1\").attr(\"id\") === \"orange-text\", 'message: Your <code>h1</code> element should have the id of <code>orange-text</code>.');",
-        "assert(code.match(/<h1[\\s\\S]*\\s+style\\s*=\\s*(?:\\'|\")\\s*(?:[\\s\\S]*;+)*\\s*color\\s*:\\s*white\\s*(?:;+[\\s\\S]*;*)*(?:\\'|\")+/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
+        "assert(code.match(/<h1[\\s\\S]*\\s+style\\s*=\\s*(?:\\'|\")\\s*(?:[\\s\\S]*;+)*\\s*color\\s*:\\s*white\\s*(?:;+[\\s\\S]*;*)*(?:\\1)+/gi), 'message: Give your <code>h1</code> element the inline style of <code>color&#58; white</code>.');",
```

Comment:
Unfortunately, I couldn't get this test to pass no matter how I tried, so I experimented a bit and came up with the following. These assertions works no matter where the code is placed and instead test that the code actually works. If we want them, they could replace the last two tests.

```js
assert(document.querySelector('h1[style]'), 'message: Give your <code>h1</code> element an inline style.');
assert((function(){var color = document.querySelector('h1[style]').style.color; return (color === 'white' || color === 'rgb(255, 255, 255)');})(), 'message: Your <code>h1</code> element should be white.');
```

This approach might be a bit easier to understand, and hopefully easier to maintain. What do you think? :blush: